### PR TITLE
add command for running scheduled tasks

### PR DESCRIPTION
**Pull request by danthe1st**
This PR adds a `/run-task` command that allows running any task annotated with `@Scheduled`.
This can be useful for development/debugging or if a task has been skipped due to e.g. downtimes.

### CODE REVIEW

File: src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		setRequiredUsers(botConfig.getSystems().getAdminConfig().getAdminUsers());
+		
+		this.taskHolder = taskHolder;
+		
+		
+	}
+	
+	@Override
+	public void execute(SlashCommandInteractionEvent event) {
+		String name = event.getOption("name").getAsString();
```
Comment (jasonlessenich) [inline src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java:51]: isnt this missing a null-check?

**Review by jasonlessenich — APPROVED** [review]

File: src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java (hunk 1 above)
Comment (danthe1st) [reply src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java:51]: The option is set to be required but I can still add a null-check.

Comment (jasonlessenich) [reply src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java:51]: yup

Comment (danthe1st) [reply src/main/java/net/discordjug/javabot/systems/staff_commands/RunScheduledTaskCommand.java]: done

**Review by MoonTM-GIT — APPROVED** [review]

**Pull request merged**